tpp-prophets/pepxml_and_protxml_to_table.xml

```diff
@@ -0,0 +1,47 @@
+<tool id="protxml_and_pepxml_to_tabular" name="PepXML and ProtXML to Table" version="1.1.1">
+    <description>Converts a pepXML/ProtXML file to a tab delimited text file</description>
```

Comment:
This tool is a combination of your previous 2 separate tools. depending on the file type is will choose the correct tool to convert.